src/Resources/app/administration/src/helper/versionHelper.js

```diff
@@ -0,0 +1,44 @@
+/*
+ *                       ######
+ *                       ######
+ * ############    ####( ######  #####. ######  ############   ############
+ * #############  #####( ######  #####. ######  #############  #############
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ *
+ * Adyen plugin for Shopware 6
+ *
+ * Copyright (c) 2023 Adyen N.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ *
+ */
+let exports = {};
+
+exports.isVersionOlderThan65 = () => {
+    function semverCompare(a, b) {
+        if (a.startsWith(b + "-")) {
+            return -1
+        } else if (b.startsWith(a + "-")) {
+            return  1
+        }
+
+        return a.localeCompare(b, undefined, { numeric: true, sensitivity: "case", caseFirst: "upper" })
+    };
+
+    const version = Shopware.Context.app.config.version;
+
+    if (semverCompare(version, "6.5.0.0") === -1) {
+        return true
+    } else {
+        return false;
+    }
```

Comment:
```suggestion
    return semverCompare(version, "6.5.0.0") === -1
```

Reply:
If the given versions are same, `semverCompare()` returns `0`. Due to this, I can't apply your suggestion because `0` will return false also, but in reality it's version `6.5.0.0`.